runtime/centrifuge/src/lib.rs

```diff
@@ -708,13 +633,13 @@ impl pallet_democracy::Config for Runtime {
 	/// A unanimous council can have the next scheduled referendum be a straight default-carries
 	/// (NTB) vote.
 	type ExternalDefaultOrigin = AllOfCouncil;
-	/// A super-majority can have the next scheduled referendum be a straight majority-carries vote.
-	type ExternalMajorityOrigin = EnsureProportionAtLeast<AccountId, CouncilCollective, 3, 4>;
+	/// A simple-majority can have the next scheduled referendum be a straight majority-carries vote.
+	type ExternalMajorityOrigin = EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 2>;
```

Comment:
I would be nice to use the `HalfOfCouncil` alias here.

Reply:
Done in https://github.com/centrifuge/centrifuge-chain/pull/1013/commits/ac82d5afa6b203dc7fc15667e4916939a2623b35